CMakeLists.txt

```diff
@@ -82,20 +85,22 @@ if (MSVC)
     add_definitions( "/W3 /D_CRT_SECURE_NO_WARNINGS /wd4996" )
 endif()
 
-list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/Modules)
-find_package(CXX11Features)
 
 if (ERT_USE_OPENMP)
-   find_package(OpenMP)
-   if (OPENMP_FOUND) 
-       message(STATUS "Enabling OpenMP support")
-       # The actual use of OpenMP is only in the libecl library - the compile flags is only applied there.
-   else()
-       set( ERT_USE_OPENMP OFF )
-       message(STATUS "OpenMP package not found - OpenMP disabled")
-   endif()
+  find_package(OpenMP)
```

Comment:
Checking the presence of OpenMP AFTER find_package(CXX11Features) fails due to the CXX flags that are set by  find_package(CXX11Features).